diff --git a/Makefile b/Makefile
index 3286f6d..9068c51 100644
--- a/Makefile
+++ b/Makefile
@@ -24,7 +24,7 @@ VCD_OBJ_FILES   = $(patsubst $(SRC_DIR)/%.cpp,$(BUILD_DIR)/%.o,$(VCD_SRC)) $(YAC
 
 TEST_APP        ?= $(BUILD_DIR)/vcd-parse
 
-all : vcd-parser docs $(BUILD_DIR)/libverilog-vcd-parser.a
+all : vcd-parser $(BUILD_DIR)/libverilog-vcd-parser.a
 
 vcd-parser: $(TEST_APP)
 
diff --git a/src/VCDParser.ypp b/src/VCDParser.ypp
index 6b3f0d4..0d8c321 100644
--- a/src/VCDParser.ypp
+++ b/src/VCDParser.ypp
@@ -152,7 +152,7 @@ declaration_command :
     if (new_signal->size == 1) {
         assert((new_signal->lindex == -1) || (new_signal->rindex == -1));
     } else {
-        assert((new_signal->lindex > 0) && (new_signal->lindex - new_signal->rindex + 1 == new_signal->size));
+        // assert((new_signal->lindex > 0) && (new_signal->lindex - new_signal->rindex + 1 == new_signal->size));
     }
     VCDScope * scope = driver.scopes.top();
     scope -> signals.push_back(new_signal);
diff --git a/src/VCDScanner.l b/src/VCDScanner.l
index 5bd1206..5a6c42d 100644
--- a/src/VCDScanner.l
+++ b/src/VCDScanner.l
@@ -394,6 +394,7 @@ void VCDFileParser::scan_begin() {
         error("Cannot open "+filepath+": "+strerror(errno));
         exit(EXIT_FAILURE);
     }
+    yypush_buffer_state(yy_create_buffer( yyin, YY_BUF_SIZE ));
 }
 
 void VCDFileParser::scan_end() {
